docs/user/tutorials/01-40-10-gcp-redis-instance.md

```diff
@@ -0,0 +1,157 @@
+# Create Redis Instances in Google Cloud
+
+Learn how to create a Redis instance in Google Cloud and connect to it.
+
```

Comment:
This tutorial does not (and should not) show how to create Redis in the Google Cloud.
It shows how to create it in Kubernetes using Kyma CRD, and as a side effect of that it will be created in the Google cloud.